### DISCUSSION TITLE
Is there a way to configure PageSize in order to return nextlink?

### DISCUSSION THREAD

**Discussion by Condor2708** [AutoMapper/AutoMapper.Extensions.OData#146] [Q&A] [answered]

**Comment by BlaiseD** [comment, marked as answer]
The examples in the [tests](https://github.com/AutoMapper/AutoMapper.Extensions.OData/blob/8b5d4896adc9404ea7ec581826b3327665062b15/AutoMapper.OData.EFCore.Tests/GetQueryTests.cs#L572) should be helpful.

**Reply by Condor2708** [reply]
Thanks @BlaiseD !
